web/src/app/schedules/temp-sched/TempSchedShiftsList.tsx

```diff
@@ -132,15 +212,66 @@ export default function TempSchedShiftsList({
           ),
         })
 
+        // prevents actions from rendering on each item if it's for the same shift
         s.added = true
+
+        // check coverage until the next shift (if there is one) within the current day
+        if (
+          shiftIdx < dayShifts.length - 1 &&
+          checkCoverage(s.end, dayShifts[shiftIdx + 1].start)
+        ) {
+          result.push({
+            render: () => (
+              <Alert
+                key={s.end.toISO() + '-no-middle-coverage'}
+                className={classes.alert}
+                severity='warning'
+              >
+                No coverage from {s.end.setZone(zone).toFormat('h:mm a')} to{' '}
+                {dayShifts[shiftIdx + 1].start.setZone(zone).toFormat('h:mm a')}
+              </Alert>
+            ),
+          })
+        }
+
+        // check end of day/temp sched coverage
+        // if on the last day, temp sched end is used
+        const _e = dayIdx === days.length - 1 ? DateTime.fromISO(end) : day.end
```

Comment:
Since the notices are for previewing the coverage for a temporary schedule, I don't think we can use `lastShiftEnd` here in any case. We specifically want to let the user know if they have all the shift coverage needed within the start/end they set on step 1.